packages/content/types/query-builder.d.ts

```diff
@@ -0,0 +1,80 @@
+interface QueryBuilderOptions {
+  query: any;
+  path: any;
+  init: any;
+  text: any;
+  postprocess?: any[];
+}
+
+export interface QueryBuilder {
+  constructor(
+    { query, path, init, text, postprocess }: QueryBuilderOptions,
+    options: any
+  );
+  query: any;
+  path: any;
+  init: any;
+  postprocess: any[];
+  options: any;
+  onlyKeys: any[];
+  withoutKeys: any;
+  sortKeys: any[];
+  limitN: number;
+  skipN: number;
+  /**
+   * Select a subset of fields
+   * @param {Array} keys - Array of fields to be picked.
```

Comment:
You don't need to provide types in JSDOC comments when using TypeScript. These might diverge down the line which could cause confusion. Also TS types are more specific when using interfaces.
Instead you can just leave the descriptions:

```typescript
/**
   * Select a subset of fields
   * @param keys - Fields to be picked
   * @returns Current instance to be chained
   */
```